ocaml/xapi/xapi.ml

```diff
@@ -467,6 +467,18 @@ let attempt_pool_hello my_ip =
         [ExnHelper.string_of_exn exn] ;
       Some Temporary
 
+let compare_xapi_version_with_coordinator ~__context =
+  let coordinator_ref = Helpers.get_master ~__context in
+  let coordinator_xapi_ver =
+    Helpers.call_emergency_mode_functions (Pool_role.get_master_address ())
```

Comment:
May I know the difference between `Helpers.call_emergency_mode_functions` and `Helpers.call_api_functions`?
This is called after one usage of `Helpers.call_api_functions` (to get the existing alerts), should it use `Helpers.call_api_functions` also?
Anyway, it's better to understand the difference between the two functions.

Reply:
Checked the code, I think the difference between them mainly in if master's DB is used:
`call_api_functions`: uses the master's database
`call_emergency_mode_functions`: uses local storage
As here the host might have not synced with master DB, I think we should only use `call_emergency_mode_functions`.